cmd/control-heal-main.go

```diff
@@ -75,14 +75,27 @@ func healControl(ctx *cli.Context) {
 	parsedURL, err := url.Parse(ctx.Args()[0])
 	fatalIf(err, "Unable to parse URL")
 
+	client, err := rpc.DialHTTPPath("tcp", parsedURL.Host, path.Join(reservedBucket, controlPath))
+	fatalIf(err, "Unable to connect to %s", parsedURL.Host)
+
+	// Always try to fix disk metadata
+	fmt.Print("Checking and healing disk metadata..")
+	args := &HealDiskMetadataArgs{}
+	reply := &HealDiskMetadataReply{}
+	err = client.Call("Control.HealDiskMetadata", args, reply)
+	fatalIf(err, "RPC Control.HealDiskMetadata call failed")
```

Comment:
oh, I see fatalIf will print the reason of the problem besides "Unable to heal disk metadata", will do it